samza-yarn/src/test/java/org/apache/samza/job/yarn/TestYarnFaultDomainManager.java

```diff
@@ -182,9 +182,17 @@ private void assertFaultDomainEquals(FaultDomain faultDomain1, FaultDomain fault
 
   private NodeReport createNodeReport(String host, int port, NodeState nodeState, String httpAddress, String rackName,
                                       int memoryUsed, int vcoresUsed, int totalMemory, int totalVcores, int numContainers,
-                                      String healthReport, long lastHealthReportTime, Set<String> nodeLabels) {
-    return NodeReport.newInstance(NodeId.newInstance(host, port), nodeState, httpAddress, rackName,
-            Resource.newInstance(memoryUsed, vcoresUsed), Resource.newInstance(totalMemory, totalVcores), numContainers,
-            healthReport, lastHealthReportTime, nodeLabels);
+                                      String healthReport, long lastHealthReportTime) {
+    return NodeReport.newInstance(
```

Comment:
Same comment here: is this change for YARN 3.3.4? If not, let's leave changes not related to JDK11 runtime support out of this PR.